### ISSUE TITLE
Invalid GraphQL query results in error when planning trip

### ISSUE THREAD

**Issue by brodyFlannigan** [opentripplanner/otp-react-redux#1131]
Hi!

I'm encountering an issue with trip planning using OTP-RR.

I'm running OTP 2.4.0 and the latest version of OTP-RR (commit [e004d3b](https://github.com/opentripplanner/otp-react-redux/commit/e004d3b355e3dd6e5618ba8552ff7818efa0f154))

The issue I'm having is that it seems that when planning a trip, the query sent to the OTP server is invalid.

Specifically, GraphQL doesn't like being asked for `numberOfTransfers` and returns the following error in it's response.

```
{
    "errors": [
        {
            "message": "Validation error (FieldUndefined@[plan/itineraries/numberOfTransfers]) : Field 'numberOfTransfers' in type 'Itinerary' is undefined",
            "locations": [
                {
                    "line": 227,
                    "column": 7
                }
            ],
            "extensions": {
                "classification": "ValidationError"
            }
        }
    ]
}
```
Using the GraphQL debugging UI (GraphiQL) on the OTP server, I've determined that removing line 227 from the GraphQL query results in a trip plan being returned.

Additionally, when this error is returned by the OTP server, no indication is provided to the user that something went wrong. They're just presented with a blank list of trip plan results.

**Comment by miles-grant-ibigroup** [comment]
Yep, there is a breaking change in OTP that caused our front-end to not work with older versions. Frustrating! We should have better error handling, we've been wanting to do that for a while. We welcome PRs!
